pcsx2/CMakeLists.txt

```diff
@@ -93,8 +93,7 @@ set(pcsx2Sources
 	MTGS.cpp
 	MTVU.cpp
 	Patch.cpp
-	BoothMultiplier.cpp
-	Ps2Float.cpp
+	PS2Float.cpp
```

Comment:
The files will also need an uppercase S for CMake to be able to find them on Unix systems

Reply:
Ps2Float.h (in addition to the now corrected PS2Float.cpp) will also need a capital S in its filename.